src/hb.hh

```diff
@@ -499,14 +499,9 @@ static_assert ((sizeof (hb_var_int_t) == 4), "");
 #define HB_VAR_ARRAY 1
 #endif
 
-static inline double
-_hb_roundf (float x)
-{
-  return x >= 0 ? floor ((double) x + .5) : ceil ((double) x - .5);
-}
-#ifndef HAVE_ROUNDF
+static inline float
+_hb_roundf (float x) { return floorf (x + .5f); }
 #define roundf(x) _hb_roundf(x)
```

Comment:
Shouldn’t this be renamed `hb_roundf` (or even `hb_otround` like FontTools, or some other name), and be used directly so that who ever reads the code/uses it in the future knows it is not the libc `roundf`?

Reply:
I thought about that. But figured we don't want someone writing code in the future to use `roundf` by mistake. There's no reason I can currently think of that we might want the default `roundf` behavior.

Reply:
All good then.

Reply:
Thanks. I let Ebrahim check bots and merge.